Tools/build.bat

```diff
@@ -76,8 +76,8 @@ for %%N in (%SUPPORTED_BUILD_SPECS%) do (
   )
 )
 
-rem Silently exit if the build target could not be matched
-goto :eof
+rem Return error if the build target could not be matched
+goto :fail
```

Comment:
The reason for ignoring build target mismatch are projects like NetKVM, viostor, and vioscsi, which build different sln/vcxproj for different targets. NetKVM for example:

```
@echo off
call ..\tools\build.bat netkvm-vs2015.sln "Wlh Win7 Win8 Win8.1 Win10" %*
if errorlevel 1 goto :eof
call ..\tools\build.bat NetKVM-VS2015.vcxproj "Win10_SDV" %*
if errorlevel 1 goto :eof
call ..\tools\build.bat NDIS5\NetKVM-NDIS5.sln "WXp Wnet" %*

```

With this change, I can no longer build NetKVM for Wnet by typing "buildall Wnet" in the NetKVM directory because the first build.bat invocation fails. And same with the top-level buildAll actually. Any chance we can still keep this working?